android-database-sqlcipher/build.gradle

```
@@ -40,7 +40,7 @@ android {
     }
 
     dependencies {
-        implementation "androidx.sqlite:sqlite:2.0.1"
+        implementation "androidx.sqlite:sqlite:2.2.0"
```

```suggestion
        implementation "androidx.sqlite:sqlite:2.1.0"
```

It looks to me like androidx.sqlite:sqlite `2.2.0` requires Android SDK 31, not sure we want to require this yet.